### PR TITLE
iris 2.4.0rc0

### PR DESCRIPTION
**Pull request by bjlittle**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bjlittle** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -7,3 +7,9 @@ travis:
   secure: {BINSTAR_TOKEN: nNvWQJrxMuAYKMLCzuq+rrrmWF0V/den+n1wJHQ7zU7ix+jbNrpp7m9SosLhBL+EhSJ62cguNtrrA9+MMO/ZJx52Lg/Cm9MTBUoCkZgyaUdrXhoqcCupXmFM2SEirIng7Dw4q+qsQSx8/7ugbphCMmp1YtM6GuY6lQYSBnTJ0NN9OjP1WlpbtFar6HGUuk9uOcjlo4gEFPAvGU+wLIZN9q3HvKMuQjvd624fCXeZnCSMULzcOaIcCq1f2aQ04zS9hL6hgxtiARMnFhykvR6bDtCJOrAP0+mg6n2Yo9jmk7UnNCLtxe+lbxtw/q5hybzjGnoLe6h4yNsdyPEfxqIVM1Wg4oz78wNdDIAKbJAtGcUc3LbyOcNFdM9DuXsIM201d4dZjAvkZtRtWxWvdQcwX1oaAPfrFFZcqBoq/pHEAQUteX7UTEjipA1zWVU4VxHFqhmXoR8zwb/2jTF75jmFDKqvf6FdbW3/1tLdIkDSVJTKxznfwAcOznosjlwrCavKYYCkM6+d4h2dX9ZCg9AtfeUWgYC46OsGYcLZAPZVEEL7p6mSmEC5Hjcjv1kuAqoluVzmEF8cdgHKwp8E9FDVOL/lUlji+hZThzg3OtQfx5UjMUIEbZ4a0SdtMhRrDJSeOyUblGx0A1J210+PSHjciI+WxlgxJ5QuKb9qoYoS78w=}
 provider:
   win: azure
+channels:
+  sources:
+    - conda-forge
+    - defaults
+  targets:
+    - ["conda-forge", "rc"]
```
Comment (bjlittle) [inline conda-forge.yml]: @ocefpaf Is this the correct way to ensure that the built release-candidate is pushed to the `conda-forge/labels/rc` channel on anaconda?

Comment (ocefpaf) [reply conda-forge.yml]: Nope. Let me update that...

Comment (ocefpaf) [reply conda-forge.yml]: Also, we'll need a dev branch for it.

**Comment by ocefpaf** [comment]
@bjlittle see https://github.com/conda-forge/cfep/blob/master/cfep-05.md for more info.

File: conda-forge.yml (hunk 1 above)
Comment (bjlittle) [reply conda-forge.yml]: Ahh nice one, thanks for clarifying :+1:

Comment (bjlittle) [reply conda-forge.yml]: @ocefpaf I noticed in cfep-05 it gives the example as `rc_matplotlib` but on the conda-forge anaconda cloud repo there is the label `matplotlib_rc`. Does that matter? Or is the protocol to just stick to the order stated in the cfep?

Also, why do some package release into the `rc` channel like `pandas`, `matplotlib` and `zarr`?

Apologies for so many questions :blush:

Comment (ocefpaf) [reply conda-forge.yml]: All of those are from before the CEP-05. Actually, `iris` is probably the first one to be CFEP-05 compliant. We have a long way ahead to fix the exiting dev releases.

Comment (bjlittle) [reply conda-forge.yml]: Ahhh cool, I understand now thanks :smile: 

Thanks for getting back to me @ocefpaf, much appreciated indeed :tada:

**Pull request merged**